seedlet/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "seedlet",
-  "version": "1.1.3",
+  "version": "1.1.4-wpcom",
```

Comment:
Is the `-wpcom` necessary in the package.json?

Reply:
I don't this so, I just thought maybe they should match?